pmedian.jl

```diff
@@ -209,9 +209,11 @@ function solve_scs_direct(data::PMedianData; max_iters)
 end
 
 function solve_moi(data::PMedianData, optimizer; params)
-    model = MOI.Bridges.full_bridge_optimizer(MOI.Utilities.CachingOptimizer(
+    cached = MOI.Utilities.CachingOptimizer(
         MOI.Utilities.UniversalFallback(MOI.Utilities.Model{Float64}()),
-        optimizer()), Float64)
+        optimizer())
+    model = MOI.Bridges.full_bridge_optimizer(cached, Float64)
+    MOI.Utilities.reset_optimizer(cached)
```

Comment:
Could you add a comment explaining why this is done?

Reply:
First attempt there